<launch>
	<!-- declare arg to be passed in -->
	<arg name="image_topic" default="/video_flow"/>

    <!-- Nodes -->
    <node name="ros_tld_gui_node" pkg="detection_tracking" type="tld_gui_node"
        output="screen">
        <remap from="image" to="$(arg image_topic)"/>
        <remap from="tracked_object" to="tld_tracked_object"/>
        <remap from="fps_tracker" to="tld_fps"/>
    </node>
</launch>

